docs/reference/commandline/volume_inspect.md

```diff
@@ -0,0 +1,24 @@
+<!--[metadata]>
++++
+title = "volume inspect"
+description = "The volume inspect command description and usage"
+keywords = ["volume, list"]
+[menu.main]
+parent = "smn_cli"
+weight=1
```

Comment:
<!--[metadata]>
+++
title = "volume inspect"
description = "The volume inspect command description and usage"
keywords = ["volume, list"]
[menu.main]
parent = "smn_cli"
+++
<![end-metadata]-->

# volume inspect

    Usage: docker volume inspect [OPTIONS] [VOLUME NAME]

    Inspect a volume

    -f, --format=       Format the output using the given go template.

Returns information about a volume. By default, this command renders all results
in a JSON array. You can specify an alternate format to execute a given template
is executed for each result. Go's
[text/template](http://golang.org/pkg/text/template/) package describes all the
details of the format.